index.js

```diff
@@ -1,10 +1,19 @@
 const express = require("express");
+const bp = require("body-parser");
```

Comment:
You dont need to use body-parser
express has the built in parser `express.json()` - you can use that instead and save adding an extra package to your app